rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +86,16 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self.args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell', add_help=False)
+        parser.add_argument('-i', '--init_script', default="",
```

Comment:
The argument is accessed using ```args.init_script```. If it was ```init-script``` it would not be a proper variable name.